kolibri/core/content/management/commands/exportcontent.py

```diff
@@ -120,6 +115,24 @@ def handle_async(self, *args, **options):
         )
         content_manifest.write(manifest_path)
 
+    def copy_content_files(self, channel_id, data_dir, files, total_bytes_to_transfer):
+        logger.info(
+            "Exporting content for channel id {} to {}".format(channel_id, data_dir)
+        )
+        exported_files = []
```

Comment:
I know you copy pasted this from the original implementation - but it seems like this exported_files value is not actually used? Feels like we could remove it and stop collecting the list if it's unused. At a minimum, it will slightly reduce memory usage.

Reply:
Done!!